backend/src/schema/resolvers/passwordReset.spec.js

```diff
@@ -15,10 +15,13 @@ let variables
 
 const getAllPasswordResets = async () => {
   const session = driver.session()
-  const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
-  const resets = transactionRes.records.map(record => record.get('r'))
-  session.close()
-  return resets
+  try {
+    const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
+    const resets = transactionRes.records.map(record => record.get('r'))
+    return resets
+  } finally {
+    session.close()
+  }
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T13:08:58Z" title="Tuesday, December 3rd 2019, 2:08:58 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -15,10 +15,13 @@ let variables
 
 const getAllPasswordResets = async () => {
   const session = driver.session()
-  const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
-  const resets = transactionRes.records.map(record => record.get('r'))
-  session.close()
-  return resets
+  try {
+    const transactionRes = await session.run('MATCH (r:PasswordReset) RETURN r')
+    const resets = transactionRes.records.map(record => record.get('r'))
+    return resets
+  } finally {
+    session.close()
+  }
```


why aren't you using `neode.cypher` here?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-03T16:35:09Z" title="Tuesday, December 3rd 2019, 5:35:09 pm +01:00">Dec 3, 2019</time>_

---


:man_shrugging: it's a test so I wouldn't mind using `neode` here. But using plain javascript driver is not a problem either, oder?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T16:37:14Z" title="Tuesday, December 3rd 2019, 5:37:14 pm +01:00">Dec 3, 2019</time>_

---


I don't thinks it's a problem, but it's more code, and I'm curious to know how `neode.cypher` handles promises/closing sessions, etc...